sdk/eventhubs/azure-messaging-eventhubs/src/private/eventhubs_utilities.hpp

```diff
@@ -265,7 +265,7 @@ namespace Azure { namespace Messaging { namespace EventHubs { namespace _detail
     {
       constexpr const char* packageName = "azure-messaging-eventhubs-cpp";
 
-      options.Properties.emplace("product", +packageName);
+      options.Properties.emplace("product", packageName);
```

Comment:
You say Go, Does this align with all languages?

Reply:
Go, .Net, and Java. After Java, I stopped looking :).

Reply:
Also python and js (checked them).

So C++ was the only SDK using upper case.